denizen_scripts/survival/repo_link/shops/claim_upgrades_shop.dsc

```diff
@@ -46,6 +43,10 @@ claiming_protection_group_upgrades_events:
   type: world
   debug: false
   events:
+    on player opens claiming_protection_groyp_upgrades:
+      - define cost <script[claiming_system_upgrade_data].parsed_key[upgrades.claim_limit.cost]>
+      - define "lore:<&a>---------------------|<&e>Price<&co><&sp><&a><[cost]>|<&b>Right click while holding.|<&b>This will unlock <&a>10 <&b>more claims.|<&a>---------------------"
+      - give o:<item[claiming_group_upgrade_item].with[material=gold_block;display_name=<&b>Upgrade<&sp>Claim<&sp>Limit;lore=<[lore]>;nbt=upgrade/claim_limit|cost/<[cost]>]> to:<context.inventory>
```

Comment:
`o:` is an invalid argument for the `give` command
it's also kinda weird to use the "args with spaces quoted" style one line above and then switch to args<&sp>without<&sp>spaces lol
here's two different suggestions i made
```suggestion
      - define "lore:<&a>---------------------|<&e>Price<&co><&sp><&a><[cost]>|<&b>Right click while holding.|<&b>This will unlock <&a>10 <&b>more claims.|<&a>---------------------"
      - give "claiming_group_upgrade_item[material=gold_block;display_name=<&b>Upgrade Claim Limit;lore=<[lore]>;nbt=upgrade/claim_limit|cost/<[cost]>]" to:<context.inventory>
```
```suggestion
      - define lore:<&a>---------------------|<&e>Price<&co><&sp><&a><[cost]>|<&b>Right<&sp>click<&sp>while<&sp>holding.|<&b>This<&sp>will<&sp>unlock<&sp><&a>10<&sp><&b>more<&sp>claims.|<&a>---------------------
      - give claiming_group_upgrade_item[material=gold_block;display_name=<&b>Upgrade<&sp>Claim<&sp>Limit;lore=<[lore]>;nbt=upgrade/claim_limit|cost/<[cost]>] to:<context.inventory>
```